### PR TITLE
Update travis with python 3.6 support

### PR DESCRIPTION
**Pull request by aojea**

### CODE REVIEW

**Comment by Itxaka** [comment]
Should probably use:

```
matrix:
  allow_failures:
    - python: '3.6'
  include:
    - python: '2.7'

```

File: psql2mysql.py [hunk 1]
```diff
@@ -72,7 +76,7 @@ def scanTablefor4ByteUtf8Char(self, table):
             for col in stringColumns:
                 if type(row[col]) is not unicode:
```
Comment (aojea) [inline psql2mysql.py]: I'm stuck in this part, because in python3 all strings are unicode

Comment (rhafer) [reply psql2mysql.py]: Using `if not isinstance(row[col], six.string_types)` should work here.

**Comment by aojea** [comment]
@rhafer I addressed your comments

**Pull request merged**